components/patient-data/impl/src/main/java/org/phenotips/data/internal/controller/GeneListController.java

```diff
@@ -271,13 +271,13 @@ private JSONObject assebleGene(Map<String, String> internalToJSONkeys, Map<Strin
             JSONObject solvedGene = json.optJSONObject(JSON_SOLVED_KEY);
 
             List<Map<String, String>> accumulatedGenes = new LinkedList<>();
-            List<String> geneSymbols = new ArrayList<>();
+            List<String> geneNames = new ArrayList<>();
 
-            parseGenesJson(genesJson, geneSymbols, accumulatedGenes, enumValues);
+            parseGenesJson(genesJson, geneNames, accumulatedGenes, enumValues);
 
             // 1.2.* json compatibility
```

Comment:
version 1.2 does not use ensemle IDs, so I think there should be a translation layer which first attempts to convert symbols to IDs (since in most cases it is possible)